spec/requests/attributes_controller_spec.rb

```diff
@@ -0,0 +1,117 @@
+RSpec.describe AttributesController do
+  before { stub_oidc_discovery }
+
+  let(:headers) { { "GOVUK-Account-Session" => placeholder_govuk_account_session } }
+
+  let(:attribute_name1) { "name" }
+  let(:attribute_value1) { { "some" => "complex", "value" => 42 } }
```

Comment:
This example would seem to suggest the string -> sym transition works pretty well